config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Configuration for running the complete pipeline including modeling and evaluation.
+
+# Absolute paths to non-code resources.
+file_paths:
+  queries: "/home/jupyter/github_msba_capstone/msba-capstone-2020/queries"
+
+# Global config used by multiple pipeline steps. Also made available
+# as parameters to all SQL queries.
+global:
+  # Project and dataset configuration.
+  source_project_id: "bigquery-public-data"
+  destination_project_id: "ut-goog"
+  source_dataset: "cfpb_complaints"
+  destination_dataset: "cfpb_complaints"
+
+  # Clean data, minor refined versions of provided data.
+  source_table: "complaint_database"
+  nlp_features_table: "complaints_nlp"
+  cleaned_features_table: "complaints_clean_categories"
+  nulls_removed_table: "complaints_nulls_removed"
+  clean_table: "complaints_clean"
+  
+  # Split features into training and prediction.
```

Comment:
Are these table names that are written? Make that clear. I'm not sure about "train_predict_split".

Reply:
Have added comments. Please review.

Reply:
If you get a chance, pop back in here at some point and add the periods to the end of those comments. But not critical.

Reply:
Actually, scratch that. I'm going to edit this directly, committing on to the branch. You will want to pull this branch before you work on it more.